### ISSUE TITLE
Add project cards from Log file 

### ISSUE THREAD

**Issue by e-lo** [wsp-sag/network_wrangler#85]
`/examples/stpaul/project cards`

**Comment by i-am-sijia** [comment]
Shall I commit them to master or develop?

**Comment by kulshresthaa** [comment]
develop

**Comment by kulshresthaa** [comment]
@e-lo so looks like this iteration of project cards aren't ready to consume? Perhaps we should pick one of them (say AddBridgetoLake.yml) and finalize how it will look like?

**Comment by e-lo** [comment]
Good idea - please do so

**Comment by kulshresthaa** [comment]
@e-lo 

a simple project card to add new roadway [here](https://github.com/wsp-sag/network_wrangler/blob/develop/example/stpaul/project_cards/AddBridgetoLake_AK.yml)

thoughts?

@i-am-sijia needs to write out the properties that are on the network. Discussed [here](https://github.com/wsp-sag/network_wrangler/commit/3fd93ac78cbe04e83c6f27822739fd42df3ddaa7#r35381889)

**Comment by e-lo** [comment]
@kulshresthaa I think this is good but for not having anything about the node locations (which I guess is OK so long as they are in the network...but let's not start with that assumption).  

The one other consideration I would make (and I'll leave this up to you to decide) is that it might make sense to use the keyword `links` (and `nodes`) instead of `properties` so that it is clearly distinguished from the editing of things.

**Comment by kulshresthaa** [comment]
yeah, I forgot to ask about the node locations that were we assuming the nodes to be already in the network and links are joining some of the existing nodes or this could include adding new nodes. 

I guess we will include nodes keywords and write out nodes in the project card if new nodes are being added?

haha.. I was using `links` but then changed it to `properties` at the last minutes to keep it consistent with other cards. Will change it back to `links` :)

**Comment by kulshresthaa** [comment]
updated.

Also, I forgot what we decided about categories - allowing multiple categories in the same project card or one for each category (with requisites etc. defined if required)? So far, the code is setup to consume one category of change in the project card.

**Comment by e-lo** [comment]
Current version looks good so long as you can add the minimum required properties to the nodes (I think it is an xy).  

One more thing to think about is node numbers. Easiest thing to do now is to assert one in the project card and just have it fail if it is already in the network (or warn if it is in the network at the same location).  Todo in future and marked in issue #93

**Comment by kulshresthaa** [comment]
but since for this case (add bridge to lake project card), no new node is added.. just two links b/w the existing nodes, we don't need the nodes properties. 
I will include the nodes properties when I define the project card for some other project which involves addition of nodes.

**Comment by e-lo** [comment]
> So far, the code is setup to consume one category of change in the project card.

Yes...and we should update it so that it can be flexible to allow:
```
project:
tags:
prerequisites:...
changes:
   - category:
      selection:
      properties:
   - category:
      selection:
      properties:
notes:
```

I created this issue in #94

**Comment by e-lo** [comment]
> no new node is added.. just two links b/w the existing nodes, we don't need the nodes properties.

Ahhh...I didn't realize that. In that case, we shouldn't require the nodes keyword (although it should be fine leaving it as well).

Just thinking ahead to possibly more complexity...we could add a `shape` keyword if that is what changed.
